.github/workflows/ci.yml

```diff
@@ -161,6 +161,13 @@ jobs:
         alpine: ${{ fromJson(needs.supported-alpine-versions.outputs.alpine) }}
         php: ${{ fromJson(needs.supported-php-versions.outputs.php) }}
         type: ${{ fromJson(needs.php-type-matrix.outputs.type) }}
+        include:
+          - php: "7.1"
+            alpine: "3.10"
```

Comment:
Are there more recent version of alpine that support `PHP 7.1`?

Reply:
No, this is the most recent version: https://hub.docker.com/_/php?tab=tags&page=1&ordering=last_updated&name=7.1-cli-alpine

We can go down to 3.4 tho :stuck_out_tongue_closed_eyes: